src/mmw/js/src/modeling/models.js

```diff
@@ -108,16 +108,23 @@ var ProjectModel = Backbone.Model.extend({
         this.set('name', newName);
     },
 
+    // Flag to prevent double POSTing of a project.
+    saveCalled: false,
+
     saveProjectAndScenarios: function() {
         if (!App.user.loggedInUserMatch(this.get('user_id'))) {
+            // Fail fast if the user can't save the project.
             return;
         }
 
-        if (!this.get('id')) {
+        if (!this.get('id') && this.saveCalled) {
```

Comment:
Consider replacing `!this.get('id')` with `this.isNew()` which is more expressive